onnxruntime/test/python/onnx_backend_test_series.py

```diff
@@ -148,19 +138,15 @@ def create_backend_test(testname=None):
                                  '^test_resize_upsample_sizes_nearest_ceil_half_pixel_cpu',
                                  '^test_resize_upsample_sizes_nearest_cpu',
                                  '^test_resize_upsample_sizes_nearest_floor_align_corners_cpu',
-                                 '^test_resize_upsample_sizes_nearest_round_prefer_ceil_asymmetric_cpu',
-                                 '^test_sequence_*',
-                                 '^test_scatter_.*',
-                                 '^test_onehot_.*',
-                                 '^test_edge_pad_cpu.*',  # test data type `int32_t` not supported yet, the `float` equivalent is covered via unit tests
-                                 '^test_reflect_pad_cpu.*'  # test data type `int32_t` not supported yet, the `float` equivalent is covered via unit tests
+                                 '^test_resize_upsample_sizes_nearest_round_prefer_ceil_asymmetric_cpu'
         ]
 
         # Example of how to disable tests for a specific provider.
         # if c2.supports_device('NGRAPH'):
         #    current_failing_tests.append('^test_operator_repeat_dim_overflow_cpu')
         if c2.supports_device('NGRAPH'):
             current_failing_tests += ['^test_clip.*',
+                                      '^test_qlinearconv_cpu.*',			
```

Comment:
This was excluded for NGraph in main.cc.

Reply:
nit: 3 tabs at end of line, and the .* wildcarding at the end isn't needed is it?

---
In reply to: [336303319](https://github.com/microsoft/onnxruntime/pull/2168#discussion_r336303319) [](ancestors = 336303319)

Reply:
Fixed tabs and removed wildcarding. This seems to have been just copied over.